src/client/dfuse/ops/readdir.c

```diff
@@ -24,240 +24,137 @@
 #include "dfuse_common.h"
 #include "dfuse.h"
 
-/* TODO: This implementation is not complete, in particular it does not
- * correctly handle calls where offset != 0, which potentially generates
- * incorrect results if the number of files is greater than can be observed in
- * one call (approx 25).
- *
- * If the filesystem is only modified from one client then the results should
- * however be correct.
- */
-
 #define LOOP_COUNT 10
 
+struct rd_data {
```

Comment:
I assume that rd stands for readdir instead of read, right?

Reply:
yea, i guess i can make it more clear.